src/test/components/TooltipMarker.test.js

```diff
@@ -67,299 +67,309 @@ describe('TooltipMarker', function() {
-            time: 19.5,
-            stack: funcNames.indexOf('nsRefreshDriver::AddStyleFlushObserver'),
+        ],
+        [
+          'Styles',
+          19,
+          {
+            type: 'tracing',
+            category: 'Paint',
+            interval: 'end',
           },
-        },
-      ],
-      [
-        'ConstructRootFrame',
-        112.5,
-        {
-          type: 'tracing',
-          category: 'Frame Construction',
-          interval: 'start',
-        },
-      ],
-      [
-        'ConstructRootFrame',
-        113.3,
-        {
-          type: 'tracing',
-          category: 'Frame Construction',
-          interval: 'end',
-        },
-      ],
-      [
-        'FileIO',
-        114,
-        {
-          type: 'FileIO',
-          startTime: 114,
-          endTime: 115,
-          source: 'PoisonIOInterposer',
-          filename: '/foo/bar',
-          operation: 'create/open',
-          cause: {
-            time: 17.0,
-            stack: funcNames.indexOf('nsRefreshDriver::AddStyleFlushObserver'),
+        ],
+        [
+          'TTFI',
+          21.4,
+          {
+            type: 'Text',
+            name: 'TTFI after 100.01ms (longTask was 100.001ms)',
+            startTime: 21.4,
+            endTime: 21.4,
           },
-        },
+        ],
+        [
+          'Log',
+          21.7,
+          {
+            type: 'Log',
+            name: 'Random log message',
+            module: 'RandomModule',
+          },
+        ],
+        [
+          'Styles',
+          20.5,
+          {
+            type: 'Styles',
+            category: 'Paint',
+            startTime: 20.0,
+            endTime: 20.5,
+            elementsTraversed: 100,
+            elementsStyled: 50,
+            elementsMatched: 10,
+            stylesShared: 15,
+            stylesReused: 20,
+            cause: {
+              time: 19.5,
+              stack: funcNames.indexOf(
+                'nsRefreshDriver::AddStyleFlushObserver'
+              ),
+            },
+          },
+        ],
+        [
+          'ConstructRootFrame',
+          112.5,
+          {
+            type: 'tracing',
+            category: 'Frame Construction',
+            interval: 'start',
+          },
+        ],
+        [
+          'ConstructRootFrame',
+          113.3,
+          {
+            type: 'tracing',
+            category: 'Frame Construction',
+            interval: 'end',
+          },
+        ],
+        [
+          'FileIO',
+          114,
+          {
+            type: 'FileIO',
+            startTime: 114,
+            endTime: 115,
+            source: 'PoisonIOInterposer',
+            filename: '/foo/bar',
+            operation: 'create/open',
+            cause: {
+              time: 17.0,
+              stack: funcNames.indexOf(
+                'nsRefreshDriver::AddStyleFlushObserver'
+              ),
+            },
+          },
+        ],
       ],
-    ]);
+      profile.meta.interval
```

Comment:
Hah, that's a huge diff for one more argument, although it is much easier to read when you ignore whitespace changes.